sensu-plugins-influxdb-metrics-checker.gemspec

```diff
@@ -3,6 +3,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 require 'date'
 require_relative './lib/sensu-plugins-influxdb-metrics-checker'
+require_relative './lib/time-management'
```

Comment:
We have done the changes and verified it by building and running the gem locally